### PR TITLE
macchina: add page

### PR DESCRIPTION
**Pull request by Rolv-Apneseth**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 6.1.8

Adding page for [macchina](https://github.com/Macchina-CLI/macchina) which is a tool similar to `neofetch` for displaying system info.

### CODE REVIEW

File: pages/common/macchina.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# macchina
+
+> Macchina is a tool which fetches and displays information about your computer.
```
Comment (blueskyson) [inline pages/common/macchina.md:3]:
```suggestion
> Display information about your computer.
```

File: pages/common/macchina.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+> More information: <https://github.com/Macchina-CLI/macchina>.
+
+- List out system information, with either default settings or those specified in your configuration file:
+
+`macchina`
+
+- Specify a custom configuration file path:
+
+`macchina --config {{path/to/configuration/file}}`
```
Comment (blueskyson) [inline pages/common/macchina.md:12]:
```suggestion
`macchina --config {{path/to/configuration_file}}`
```

**Review by blueskyson — COMMENTED** [review]
Thanks for your contribution. Here are some suggestions.

File: pages/common/macchina.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- List out system information, but lengthen uptime, shell and kernel output:
+
+`macchina --long-uptime --long-shell --long-kernel`
+
+- Check for any errors / system failures encountered when trying to fetch system information:
```
Comment (kbdharun) [inline pages/common/macchina.md:18]:
```suggestion
- Check for any errors/system failures encountered when trying to fetch system information:
```

File: pages/common/macchina.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`macchina --doctor`
+
+- List out original artists of all the ASCII art used by macchina:
```
Comment (kbdharun) [inline pages/common/macchina.md:22]:
```suggestion
- List out original artists of all the ASCII art used by `macchina`:
```
Or
```suggestion
- List out original artists of all the ASCII art:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by Rolv-Apneseth** [comment]
> Thanks for your contribution. I have some suggestions for this page.

No problem and thanks, I have applied those

**Comment by Rolv-Apneseth** [comment]
> Thanks for your contribution. Here are some suggestions.

Thanks, applied those too

**Review by acuteenvy — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

File: pages/common/macchina.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- List out system information, with either default settings or those specified in your configuration file:
+
+`macchina`
+
+- Specify a custom configuration file path:
+
+`macchina --config {{path/to/configuration_file}}`
+
+- List out system information, but lengthen uptime, shell and kernel output:
```
Comment (blueskyson) [inline pages/common/macchina.md:14]: This sounds more natural.
```suggestion
- List system information, but lengthen uptime, shell and kernel output:
```

File: pages/common/macchina.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+`macchina --long-uptime --long-shell --long-kernel`
+
+- Check for any errors/system failures encountered when trying to fetch system information:
+
+`macchina --doctor`
+
+- List out original artists of all the ASCII art:
```
Comment (blueskyson) [inline pages/common/macchina.md:22]:
```suggestion
- List original artists of all the ASCII art:
```

**Review by kbdharun — APPROVED** [review]

**Pull request merged**